packages/host/app/components/operator-mode.gts

```diff
@@ -271,6 +274,24 @@ export default class OperatorMode extends Component<Signature> {
       `);
   }
 
+  @action async add() {
+    (this.addCard as unknown as Descriptor<any, any[]>).perform();
+  };
+
+  private addCard = restartableTask(async () => {
+    let type = identifyCard(this.args.firstCardInStack.constructor) ?? baseCardRef;
```

Comment:
I'm not sure here, does add card button allow user to create new card or only choose an existing card instance?

Reply:
It should tie into search to allow the user to open an existing card. That can be a separate task from this though.